tier2/{{cookiecutter.project_slug}}/README.md

```diff
@@ -4,8 +4,15 @@
 ## About the Project
 **{project statement}**
 
-## Core Team
-An up-to-date list of core team members can be found in [MAINTAINERS.md](MAINTAINERS.md). At this time, the project is still building the core team and defining roles and responsibilities. We are eagerly seeking individuals who would like to join the community and help us define and fill these roles.
+### Project Vision
+**{project vision}**
+
+#### Agency Mission
```

Comment:
Is this hierarchy intentional? Vision part of core team, with Agency Mission and team mission under agency mission?

Perhaps we can try rearranging the order of headings based on broadness>specificness, or grouping related sections? Something like:

```
## About the Project
### Project Vision
### Project Mission (is this redundant? I know there's a difference between vision/mission statements, but I am not sure how many teams/projects have/will have them.)
## Team Mission
## Core Team
```

Reply:
Order confirmed, returning Core Team section